GitVersionCore/Configuration/Init/BuildServer/AppVeyorSetup.cs

```diff
@@ -0,0 +1,122 @@
+    using GitVersion.Helpers;
+
+    class AppVeyorSetup : ConfigInitWizardStep
+    {
+        protected override StepResult HandleResult(string result, Queue<ConfigInitWizardStep> steps, Config config, string workingDirectory, IFileSystem fileSystem)
+        {
+            switch (result)
+            {
+                case "0":
+                    steps.Enqueue(new EditConfigStep());
+                    return StepResult.Ok();
+                case "1":
+                    GenerateBasicConfig(workingDirectory, fileSystem);
+                    steps.Enqueue(new EditConfigStep());
+                    return StepResult.Ok();
+                case "2":
+                    GenerateNuGetConfig(workingDirectory, fileSystem);
+                    steps.Enqueue(new EditConfigStep());
+                    return StepResult.Ok();
+            }
+            return StepResult.InvalidResponseSelected();
+        }
+
+        void GenerateBasicConfig(string workingDirectory, IFileSystem fileSystem)
+        {
+            WriteConfig(workingDirectory, fileSystem, @"install:
+  - choco install gitversion.portable -pre
+
+before_build:
+  - nuget restore
+  - ps: gitversion /l console /output buildserver /updateAssemblyInfo
+
+build:
+  project: <your sln file>");
+        }
+
+        void GenerateNuGetConfig(string workingDirectory, IFileSystem fileSystem)
+        {
+            WriteConfig(workingDirectory, fileSystem, @"install:
+  - choco install gitversion.portable -pre -y
+
+assembly_info:
+  patch: false
+
+before_build:
+  - nuget restore
+  - ps: gitversion /l console /output buildserver /updateAssemblyInfo
+
+build:
+  project: <your sln file>
+
+after_build:
+  - cmd: ECHO nuget pack <Project>\<NuSpec>.nuspec -version ""%GitVersion_NuGetVersion%"" -prop ""target=%CONFIGURATION%""
+  - cmd: nuget pack <Project>\<NuSpec>.nuspec -version ""%GitVersion_NuGetVersion%"" -prop ""target=%CONFIGURATION%""
+  - cmd: appveyor PushArtifact ""<NuSpec>.%GitVersion_NuGetVersion%.nupkg""");
+        }
+
+        void WriteConfig(string workingDirectory, IFileSystem fileSystem, string configContents)
+        {
+            var outputFilename = GetOutputFilename(workingDirectory, fileSystem);
+            fileSystem.WriteAllText(outputFilename, configContents);
+            Logger.WriteInfo(string.Format("AppVeyor sample config file written to {0}", outputFilename));
+        }
+
+        protected override string GetPrompt(Config config, string workingDirectory, IFileSystem fileSystem)
+        {
+            var prompt = new StringBuilder();
+            if (AppVeyorConfigExists(workingDirectory, fileSystem))
+            {
+                prompt.AppendLine("GitVersion doesn't support modifying existing appveyor config files. We will generate appveyor.gitversion.yml instead");
+                prompt.AppendLine();
+            }
+
+            prompt.Append(@"What sort of config template would you like generated?
+
+0) Back
```

Comment:
This is a general comment, but I looked at this option for a while before realising that it means "Go back".  I think the addition of the "Go" would help a lot.  Or perhaps I am just slow :smile_cat: